docs/en/sql-reference/sql-statements/data-manipulation/CREATE_ROUTINE_LOAD.md

```diff
@@ -210,6 +213,30 @@ If `property.group.id` is not specified, StarRocks generates a random value base
     "property.sasl.password" = "xxxxxx"
     ```
 
+  - Access Kafka using the SASL_PLAINTEXT security protocol and SASL/GSSAPI (Kerberos) authentication mechanism:
+
+    ```sql
+    "property.security.protocol" = "SASL_PLAINTEXT", -- Specifies the security protocol as SASL_PLAINTEXT
+    "property.sasl.mechanism" = "GSSAPI", -- Specifies the SASL authentication mechanism as GSSAPI. Default value is GSSAPI.
+    "property.sasl.kerberos.service.name" = "kafka", -- Specifies the broker service name, default is Kafka.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/710fd479-4627-4e48-af44-27b4e88a4360)
-> Default value is xxx.